src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/entities/PrConfigCommandArguments.java

```diff
@@ -0,0 +1,58 @@
+package io.sealights.plugins.sealightsjenkins.buildsteps.cli.entities;
+
+import hudson.util.DescribableList;
+import io.sealights.plugins.sealightsjenkins.buildsteps.cli.configurationtechnologies.TechnologyOptions;
+import io.sealights.plugins.sealightsjenkins.buildsteps.cli.configurationtechnologies.TechnologyOptionsDescriptor;
+
+public class PrConfigCommandArguments extends ConfigCommandArguments {
+    private String latestCommit;
+    private String pullRequestNumber;
+    private String repoUrl;
+    private String targetBranch;
+
+    public PrConfigCommandArguments(DescribableList<TechnologyOptions, TechnologyOptionsDescriptor> techOptions,
+     String latestCommit, String pullRequestNumber, String repoUrl, String targetBranch) {
+        super(techOptions);
+        this.latestCommit = latestCommit;
+        this.pullRequestNumber = pullRequestNumber;
+        this.repoUrl = repoUrl;
+        this.targetBranch = targetBranch;
+    }
+
+    public String getLatestCommit() {
+        return latestCommit;
+    }
+
+    public void setLatestCommit(String latestCommit) {

```

Comment:
Can't we use lombok for these classes?

Reply:
No its used in jelly files, tried before and didn't work